pkg/syncer/util.go

```diff
@@ -196,3 +202,16 @@ func migratedToAnnotationExists(ctx context.Context, annotations map[string]stri
 	log.Debugf("%v annotation not found", common.AnnMigratedTo)
 	return false
 }
+
+// GetSCNameFromPVC gets name of the storage class from provided PVC
+func GetSCNameFromPVC(pvc *v1.PersistentVolumeClaim) (string, error) {
```

Comment:
Do you PSP specific common util files in the project? We can move this function there, as this looks more specific to PSP?